### ISSUE TITLE
Veien videre...

### ISSUE THREAD

**Issue by Starefossen** [iojs/iojs-no#25]
Tenkte vi kunne diskutere litt veien videre. Hadde vært utrolig greit å få noen kick-ass io.js guider av ulike vanskelighetsgrad på plass: "Slik kommer du i gang", "Fra Node.JS til io.js", kanskje se litt på de ulike ES6 funksjonene som er tilgjengelig i io.js?

Kanskje noen kunne laget en enkel standard-presentasjon av io.js som andre kunne benytte hvis de har anledning til å si noen ord om io.js på en meetup eller i andre sammenhenger?

Vi må tenke litt større enn bare de som er kjent med Node i dag til de som er helt ferske og har lyst å prøve seg på JavaScript programmering. Bare kom sleng ut ideene her og hvis det er noe som virker interessant er det bare å lage nye issues eller PR for å ta diskusjonen videre.

**Comment by asbjornenge** [comment]
:+1: for en god standard-presentasjon av io på norsk. Core konsepter som eventloop & async programmering er ting vi kanskje tar for gitt, men som for mange er fjernt.

Når det gjelder "slik kommer du igang (med) ..." guider så er jeg veldig fan av en kjapp forklaring + lenke til en god workshopper. Gjordt mye bra arbeid i nodeschool leiren rundt nettop dette. Ingenting som er å prøve ting ut i "praksis". :wink: Er tanken å bygge opp er samling slike på nettsidene?

**Comment by eiriksm** [comment]
+1 for workshoppers. Det er vel minst 3-4 stk som går på es6 spesifikke ting.

nodeschool sida er jo forsåvidt mulig å oversette til norsk også. Det kunne kanskje hjulpet?

**Comment by asbjornenge** [comment]
@eiriksm det hadde vært en kjempe ide! de har jo en sånn localization effort de også...!?

**Comment by leftieFriele** [comment]
å lage norske versjoner av NodeSchool ting ville vært veldig kult, det
ville også kunne brukes i undervisning på skoler, høyskoler og
universiteter.

On Tue, Feb 10, 2015 at 9:59 AM, Asbjorn Enge notifications@github.com
wrote:

> @eiriksm https://github.com/eiriksm det hadde vært en kjempe ide! de
> har jo en sånn localization effort de også...!?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/iojs/iojs-no/issues/25#issuecomment-73665128.

## 

## <quote goes here>

Espen Dalløkken

**Comment by Starefossen** [comment]
+1 for workshoppers på norsk. Det er absolutt ikke slik at vi skal finne opp hjule på nytt men heller prøve å linke sammen godt materiale som finnes og supplere der det ikke eksisterer slik at det blir enkelt for nye (og viderekommende) og komme i gang og å finne det de er ute etter.

Post gjerne lenker til materiale som kan egne seg som utgangspunkt.

**Comment by asbjornenge** [comment]
:+1: For en slags "oppdag node" guide med lenker til workshoppers og andre ting vi/andre har oversatt... Oh, nå fikk jeg lyst å lage et spill :stuck_out_tongue_closed_eyes: mind running away :space_invader:

**Comment by zrrrzzt** [comment]
NodeSchool/workshopper på norsk hadde vært kjempemessig, men det er kanskje litt utenfor hva iojsno skal drive med?
At flere av oss her inne bidrar til oversetting og at det linkes til ressursene støtter jeg fullt ut.

**Comment by phillipj** [comment]
Yo! Fikk tak i nodejs.no for en måned siden.. Kanskje vi kunne ha gjort noe kult sammen? https://github.com/nodejs-no/www/issues/1

**Comment by Starefossen** [comment]
@phillipj så bra, bare sett opp en videresending til iojs.no du :wink: 

Men fra spøk til alvor, det hadde vært utrolig bra hvis vi kunne klart å etablere et norsk javascript/node miljø!

**Comment by Andersos** [comment]
Det finnes jo flere miljø -> http://www.meetup.com/NodeJSOslo/ http://www.meetup.com/framsia/

**Comment by Starefossen** [comment]
Det er mer enn bare Oslo i Norge ;)

**Comment by Andersos** [comment]
Trodde ikke det var nødvendig å liste opp alle http://bartjs.github.io/ http://www.meetup.com/BergenJS/

**Comment by Starefossen** [comment]
Yes, masse lokale meetups! Veldig bra, vi burde samle alle og publisere en liste :)

**Comment by snostorm** [comment]
> "Getting Started", "From Node.JS to io.js" perhaps see a bit of the various ES6 features available in io.js?

Pardon my use of English: the website WG is _very open_ to new articles being created for distribution across the iojs.org website that aren't necessary written English-first with the goal that the might be presented to the other translation groups for their translation.

On my goals (near future) is to get an articles section up, but first, we need articles to link to -- regardless of language :)
